src/main/java/com/kyonggi/teampu/domain/admin/service/AdminService.java

```diff
@@ -0,0 +1,38 @@
+package com.kyonggi.teampu.domain.admin.service;
+
+import com.kyonggi.teampu.domain.admin.dto.response.AdminAppliedInfoResponse;
+import com.kyonggi.teampu.domain.application.domain.Application;
+import com.kyonggi.teampu.domain.application.repository.ApplicationRepository;
+import jakarta.transaction.Transactional;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class AdminService {
+
+    private final ApplicationRepository applicationRepository;
+
+    public List<AdminAppliedInfoResponse> getHome(){
+        List<Application> applicationList = applicationRepository.findAllBy();
+
+        List<AdminAppliedInfoResponse> adminAppliedInfoResponseList = new ArrayList<>();
+
+        for (Application application : applicationList) {
+            adminAppliedInfoResponseList.add(new AdminAppliedInfoResponse(
+                application.getMember().getName(), application.getAppliedDate(), application.getStatus()
+            ));
+        }
+
+        return adminAppliedInfoResponseList;
```

Comment:
```suggestion
        return applicationList.stream()
            .map(application -> new AdminAppliedInfoResponse(
                application.getMember().getName(),
                application.getAppliedDate(),
                application.getStatus()
            ))
            .toList();
```

요렇게 Stream문으로 변경하면 좀 더 직관적일 것 같습니다. 

그리고 AdminAppliedInfoResponse DTO 내부에 Application 엔티티를 DTO로 변환하는 메서드를 구현하면 훨씬 간단하게 축약할 수 있을 것 같습니다!